lib/article-tile/index.js

```diff
@@ -2,7 +2,7 @@ import React, { Component, PropTypes } from 'react';
 
 require('./style.css');
 
-export class ArticleTile extends Component {
+export default class ArticleTile extends Component {
```

Comment:
what does `default` do here? 
I *thought* `default` was only necessary when the function/method/class being exported was anonymous... or when you have more than one exported method/variable ...?